pkg/services/contexthandler/contexthandler.go

```diff
@@ -50,6 +50,7 @@ const (
 const ServiceName = "ContextHandler"
 
 func ProvideService(cfg *setting.Cfg, tokenService auth.UserTokenService, jwtService jwt.JWTService,
+func ProvideService(cfg *setting.Cfg, tokenService auth.UserTokenService, jwtService models.JWTService,
```

Comment:
Bad merge?

Reply:
Sorry the noise, I need to do multiple changes with all the feedback, so since this has been paused for some weeks, I just pulled latest changes (didn't pay much attention, honestly).

I'll ping you back once ready for a final review! Thanks!